testing/functional_tests/test_segment_volume.py

```diff
@@ -144,4 +144,4 @@ def test_segment_volume_3d(download_data_testing_test_files):
 
 
 def teardown_function():
-    remove_tmp_dir()
+    remove_tmp_dir()
```

Comment:
@konantian PEP8 requires new line at the end of .py file. Could you correct that minor issue and everything else looks good!

Reply:
Thanks for your suggestion, I have fixed this issue and pushed the fix commit.